wgengine/monitor/monitor_darwin.go

```diff
@@ -87,9 +89,44 @@ func (m *darwinRouteMon) Receive() (message, error) {
 }
 
 func (m *darwinRouteMon) skipMessage(msg route.Message) bool {
-	switch msg.(type) {
+	switch msg := msg.(type) {
 	case *route.InterfaceMulticastAddrMessage:
 		return true
+	case *route.InterfaceAddrMessage:
+		return m.skipInterfaceAddrMessage(msg)
+	case *route.RouteMessage:
+		return m.skipRouteMessage(msg)
+	}
+	return false
+}
+
+func (m *darwinRouteMon) skipInterfaceAddrMessage(msg *route.InterfaceAddrMessage) bool {
+	for i, a := range msg.Addrs {
+		switch i {
+		case unix.RTAX_IFP:
+			if la, ok := a.(*route.LinkAddr); ok {
+				baseName := strings.TrimRight(la.Name, "0123456789")
+				switch baseName {
+				case "llw", "awdl", "pdp_ip":
+					return true
+				}
+			}
+		}
+	}
+	return false
+}
+
+func (m *darwinRouteMon) skipRouteMessage(msg *route.RouteMessage) bool {
+	// Skip those like:
+	// dst = fe80::b476:66ff:fe30:c8f6%15
+	for i, a := range msg.Addrs {
+		switch i {
+		case unix.RTAX_DST:
```

Comment:
Can you add a link to man pages or something in a comment here? The magical "array offsets represent different objects" thing confused us a bunch until we figured it out.